RoboFile.php

```diff
@@ -26,14 +26,14 @@ public function wordpressSetup( $opts = [
 			return;
 		}
 
-		$this->_exec( "mysql -uroot -e 'create user " . $opts['wp-db-name'] . "'" );
-		$this->_exec( "mysql -uroot -e 'create database " . $opts['wp-db-name'] . "'" );
-		$this->_exec( 'mysql -uroot -e "grant all privileges on ' . $opts['wp-db-name']
+		$this->_exec( "mysql -uroot -proot -h 0.0.0.0 -e 'create user if not exists " . $opts['wp-db-name'] . "'" );
+		$this->_exec( "mysql -uroot -proot -h 0.0.0.0 -e 'create database if not exists " . $opts['wp-db-name'] . "'" );
+		$this->_exec( 'mysql -uroot -proot -h 0.0.0.0 -e "grant all privileges on ' . $opts['wp-db-name']
 		. ' . * to ' . $opts['wp-db-name'] . "@localhost identified by '" . $opts['wp-db-name'] . "'\"" );
-		$this->_exec( "mysql -uroot -e 'flush privileges'" );
+		$this->_exec( "mysql -uroot -proot -h 0.0.0.0 -e 'flush privileges'" );
 
-		$this->wp( 'core download --version=4.8.2 --force' );
-		$this->wp( 'core config --dbname=' . $opts['wp-db-name'] . ' --dbuser=' . $opts['wp-db-name'] . ' --dbpass=' . $opts['wp-db-name'] . ' --dbhost=127.0.0.1' );
+		$this->wp( 'core download --version=4.8.2 --locale=en_GB --force' );
```

Comment:
Could we make this locale=en_US? And then .gitignore `wordpress/wp-content/languages/`?

Reply:
There is a documented error with the en_US locale here  https://github.com/wp-cli/wp-cli/issues/4370 which is why I used the en_GB. I couldn't find a way to resolve the issue without choosing another locale.

Reply:
Ah, I see! okay then, `en_GB` it is.